code/modules/modular_computers/file_system/programs/atmosscan.dm

```diff
@@ -57,7 +57,7 @@
 	var/list/data = get_header_data()
 	var/turf/turf = get_turf(computer)
 	data["atmozphereMode"] = atmozphere_mode
-	data["clickAtmozphereCompatible"] = computer.hardware_flag == PROGRAM_TABLET
+	data["clickAtmozphereCompatible"] = (computer.hardware_flag & PROGRAM_TABLET)
```

Comment:
There's no limit to who can download the app, so if I do change it, it would make other machines atmozphere compatible, even if they can't do the actions needed to trigger them.